app/gui/src/dashboard/hooks/searchParamsStateHooks.ts

```diff
@@ -13,7 +13,7 @@ import * as lazyMemo from '#/hooks/useLazyMemoHooks'
 import * as safeJsonParse from '#/utilities/safeJsonParse'
 import { useRouterInReact } from '$/providers/react'
 import { type RouteLocationOptions } from 'vue-router'
-import { ZodSchema, type z } from 'zod'
+import { type z } from 'zod'
```

Comment:
`import { type }` -> `import type {}`? same with the import from `vue-router` above